### PR TITLE
[s]CTF Christmas

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Security tag cause I don't even know if/how well it works yet and I want to test merge and test with a handful of people before getting everyones hopes up

### CODE REVIEW

**Comment by WJohn** [comment]
Grinch's versus Santa's little helpers?

**Comment by KazeEspada** [comment]
It should be red vs green, not red vs blue due to tradition.

We need radios for each team.

Capturing a flag should be captured at the flag not at the respawner.

Messages are world wide.

No cameras with "thunderdome" network(thats the entertainment network). This would allow the crew to watch.

File: code/modules/awaymissions/mission_code/capture_the_flag.dm [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+	ears = /obj/item/device/radio/headset/syndicate/alt
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf/red
+
+/datum/outfit/ctf/blue
+	ears = /obj/item/device/radio/headset/headset_cent/commander
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf/blue
+
+/datum/outfit/ctf/red/post_equip(mob/living/carbon/human/H)
+	var/obj/item/device/radio/R = H.ears
+	R.set_frequency(SYND_FREQ)
```
Comment (KazeEspada) [inline code/modules/awaymissions/mission_code/capture_the_flag.dm]: Can't use this frequency because if the round is nuke flops, nuke flops need this.

**Comment by Alek2ander** [comment]
All clothing should be NODROP, otherwise you'll have blodi spais.
Either the picked up/dropped announcements should give the area name or there need to be pinpointers set to both flags available for each team.

**Comment by KorPhaeron** [comment]
Christmas is over, ctf is kill

**Pull request closed without merging**